content/en/continuous_integration/tests/java.md

```diff
@@ -69,18 +57,31 @@ Additionally, configure which [Datadog site][2] to which you want to send data.
 
 [1]: https://app.datadoghq.com/organization-settings/api-keys
 [2]: /getting_started/site/
+
 {{% /tab %}}
+{{% tab "On-Premises CI provider (Datadog Agent)" %}}
+
+If you are running tests on an on-premises CI provider such as Jenkins or self-managed GitLab CI, you can install the Datadog Agent on each worker node by following the [Agent installation instructions][1]. Datadog recommends running CI Visibility with Agentless mode.
```

Comment:
I wonder if we can somehow make it more obvious to the customers that it is possible to use Agentless mode both with cloud CI providers and with on-premise providers, i.e. they do not need to migrate from on-premise CI to cloud CI if they want to switch from using the Agent to Agentless.

I thought that "Cloud CI provider (Agentless)" tab name was a bit misleading in this sense, so I renamed it to "Agentless", but maybe there is a better way to convey it.

@alai97 what do you think?

Reply:
@nikita-tkachenko-datadog My intention was to keep the tabs consistent across the Test pages (such as for [.NET](https://docs.datadoghq.com/continuous_integration/tests/dotnet/)).

```suggestion
If you are running tests on an on-premises CI provider such as Jenkins or self-managed GitLab CI, you can install the Datadog Agent on each worker node by following the [Agent installation instructions][1]. You can also run CI Visibility on an on-premises CI provider with Agentless mode.
```